server.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+const net = require("net");
+const files = require("./files.js");
+
+const PORT = 8080;
+
+const server = net.createServer(socket => {
+  socket.on("data", chunk => {
+    // read incoming data
+    console.log("data");
+    console.log(chunk.toString());
+
+    // parse the string
+    var test = chunk.toString().split(" ");
+    // grab the right file
+
+    // write outgoing data
+    if (test[1] === `/`) {
+      socket.write(`HTTP/1.1 200 OK
+Server: dope magic
+Content-Type: text/html
+Connection: keep-alive
+
+${files.index}`);
+    } else if (test[1] === `/index.html`){
+      socket.write(`HTTP/1.1 200 OK
+Server: dope magic
+Content-Type: text/html
+Connection: keep-alive
+
+      ${files.index}`)
+    }else if (test[1] === `/hydrogen.html`){
+      socket.write(`HTTP/1.1 200 OK
+Server: dope magic
+Content-Type: text/html
+Connection: keep-alive
+
+      ${files.hydrogen}`)
+    }else if (test[1] === `/helium.html`){
+      socket.write(`HTTP/1.1 200 OK
+Server: dope magic
+Content-Type: text/html
+Connection: keep-alive
+
+      ${files.helium}`)
+    }else if(test[1] === `/css/styles.css`){
+      socket.write(`HTTP/1.1 200 OK
+Server: dope magic
+Content-Type: text/html
+Connection: keep-alive
+
+      ${files.styles}`)
+    }else{
+      socket.write(`HTTP/1.1 404 NOT FOUND
+Server: dope magic
+Content-Type: text/html
+Connection: keep-alive
+
+      ${files.notFound}`)
+    }
+    socket.end();
+  });
```

Comment:
Missing Content-length for the headers. And missing serving out the css file as well.